src/utilities/widgets.py

```diff
@@ -191,3 +195,27 @@ class MyForm(forms.Form):
         if you just drop in the widget for a ContentObjectChoiceField field.
     """
     pass
+
+
+class SummernoteInplaceSafeWidget(SummernoteWidgetBase):
+
+    def value_from_datadict(self, data, files, name):
```

Comment:
This method will need a test and a docstring.

For docstring, please state the reason why the method is being overridden.  i.e. what problem is being solved that the original method does not handle.